decomposed/extensions/MDDE_LDM/Profile/Model/Methods/mdde_InitializeExportExtension.xml

```diff
@@ -0,0 +1,99 @@
+<o:MethodTargetItem Id="A2DF8843-2E87-46FC-9263-927B3E02D9E8">
+<a:ObjectID>A2DF8843-2E87-46FC-9263-927B3E02D9E8</a:ObjectID>
+<a:Name>mdde_InitializeExportExtension</a:Name>
+<a:CreationDate>1701423896</a:CreationDate>
+<a:Creator>WillemOtten</a:Creator>
+<a:TargetCategory.Type>1</a:TargetCategory.Type>
+<a:MethodScript>Sub %Method%(obj)
+   &#39;Implement your method on &lt;obj&gt; here
+   &#39;Set success indicator to true
+   WriteInfo &quot;MDDE LDM Export extension initializion started&quot;
+   obj.setExtendedAttribute cMDDE_EXT_CODE &amp; &quot;.&quot; &amp; cMDDE_EXPORT_EXTENSION_INITIALIZATION_INDICATOR, True
```

Comment:
If the meaning of the indicator is whether the initialization is succesfull, why don't you set it at the end?

Reply:
Valid remark, changed icw errorhandling